atest/robot/libdoc/html_output.robot

```diff
@@ -3,6 +3,14 @@ Resource          libdoc_resource.robot
 Suite Setup       Run Libdoc And Parse Model From HTML    ${TESTDATADIR}/module.py
 Test Template     Should Be Equal Multiline
 
+*** Keywords ***
+Verify Argument Models
+    [Arguments]    ${arg_models}    @{expected_reprs}
+    Should Be True    len($arg_models) == len($expected_reprs)
+    FOR    ${arg_model}    ${expected_repr}    IN ZIP    ${arg_models}    ${expected_reprs}
+       Run Keyword And Continue On Failure    Verify Argument Model    ${arg_model}    ${expected_repr}
+    END
```

Comment:
We've used order Settings/Variables/Tests/Keywords in our data files. Would be better to follow that convention here as well.